rpm/dovecot-ceph-plugin.spec

```diff
@@ -108,13 +140,13 @@ find %{buildroot}%{_libdir}/dovecot/ -type f -name \*.a  -delete
 
 %files -n librmb-devel
 %defattr(-,root,root)
-%{_includedir}/*
+%dir %{_includedir}/dovecot-ceph-plugin
+%{_includedir}/dovecot-ceph-plugin/*.h
 %{_libdir}/librmb.so
 
 %files -n rmb-tools
 %defattr(-,root,root)
-%attr(0755, root, root) %{_bindir}/rmb
-%doc %{_mandir}/man1/rmb.1*
+%{_bindir}/rmb
+/usr/share/man/man1/rmb.1.gz
```

Comment:
Please, do not use absolute path names. We have to relocate them using the macros.